rfcs/convergence/authoring-stories.md

```diff
@@ -267,13 +271,144 @@ TBA
+2. If putting all stories into one file would make it too long, individual stories might be put into additional `.stories.tsx` files as a named export, and then re-exported from `Component.stories.tsx` file like this: `export * from ‘./IndividualStoryFile.stories’;`
+3. If individual story files are employed, `Component.stories.tsx` file must not contain any stories besides the default export.
+
+
+##### Good Example 1 - single file
+
+Button.stories.tsx:
+```tsx
+import { Meta } from '@storybook/react';
+
+export const Default = (props: ButtonProps) => <Button {...props}>Button</Button>;
+export const ButtonWithIcon = () => <Button icon={<CalendarIcon />}>Text</Button>;
+
+export default {
+  title: 'Components/Button',
+  component: Button
+} as Meta;
+```
+
+##### Good Example 2 - multiple files
+
+Button.stories.tsx:
+```tsx
+import { Meta } from '@storybook/react';
+
+export * from "ButtonDefault.stories"
+export * from "ButtonWithIcon.stories"
+
+export default {
+  title: 'Components/Button',
+  component: Button
+} as Meta;
+```
+
+ButtonDefault.stories.tsx:
+```tsx
+export const ButtonDefault = (props: ButtonProps) => <Button {...props}>Button</Button>;
+ButtonDefault.storyName = "Default"
+```
+
+ButtonWithIcon.stories.tsx:
+```tsx
+export const ButtonWithIcon = () => <Button icon={<CalendarIcon />}>Text</Button>;
+```
+
+##### Bad example - multiple files
+
+ButtonDefault.stories.tsx:
+```tsx
+export const ButtonDefault = (props: ButtonProps) => <Button {...props}>Button</Button>;
+ButtonDefault.storyName = "Default"
+
+// don’t do this
+export default {
+  title: 'Components/Button',
+  component: Button
+} as Meta;
+```
+
+ButtonWithIcon.stories.tsx:
+```tsx
+export const ButtonWithIcon = () => <Button icon={<CalendarIcon />}>Text</Button>;
+
+// don’t do this
+export default {
+  title: 'Components/Button',
+  component: Button
+} as Meta;
+```
+
+### 8. location and naming convention
+
+1. Story files should be in the same folder as the component which they are meant to show off. This will usually be the component which is the most unique to the story.
+2. Name of every .stories.tsx file should start with name of a component which it is targeting, to improve colocation in alphabetical file ordering. If appropriate, story name should be adjusted via configuration to a well readable name, suitable for documentation, like this:
+
+```tsx
+export const ButtonPrimary = (props: ButtonProps) => <Button {...props}>Text</Button>;
+ButtonPrimary.args = {
+  primary: true,
+};
+ButtonPrimary.storyName = 'Better story name';
+```
+
+### 9. UX of stories
+
+#### default story
+
+Every component must have a story called `Default`, which:
+- must be the first story in the component - see [chapter 7](#7-dissecting-big-story-files-into-smaller-ones) for examples
```

Comment:
**suggestion:** must be defined as first story inline or as first re-expored story when using multi-file pattern...